# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: config/system-headers.mozbuild
# Commit: 74fa25299c53
# Full Hash: 74fa25299c53a238c11fa559f0efc7038518b5c2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 16:14:38
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r cff6a02ad4ee -r 74fa25299c53 config/system-headers.mozbuild
--- a/config/system-headers.mozbuild	Wed Feb 28 12:50:26 2024 +0000
+++ b/config/system-headers.mozbuild	Wed Feb 28 12:50:26 2024 +0000
@@ -1267,11 +1267,6 @@
         "jpeglib.h",
     ]
 
-if CONFIG["MOZ_LIBAV_FFT"]:
-    system_headers += [
-        "libavcodec/avfft.h",
-    ]
-
 if CONFIG["MOZ_SYSTEM_PNG"]:
     system_headers += [
         "png.h",